vendor/engines/secure_rooms/spec/features/edit_indala_number_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.describe "User's Indala Number" do
+  let(:facility) { FactoryGirl.create(:facility) }
+  let(:indala_user) { FactoryGirl.create(:user) }
+  let(:staff_user) { FactoryGirl.create(:user, :staff, facility: facility) }
+
+  describe "update" do
+    before do
+      login_as staff_user
+      visit facility_user_path(facility, indala_user)
+      click_link "Update"
```

Comment:
Minor thing: is there a way to scope this finder down? It seems like this test is somewhat brittle because it could break if there were another "Update" link on the page. If there's not an easy way, it's fine.